src/core/Entry.cpp

```diff
@@ -784,3 +789,20 @@ QString Entry::resolvePlaceholder(const QString& str) const
 
     return result;
 }
+
+QString Entry::resolveUrl(const QString& url) const
+{
+    QStringList parts = url.toLower().split("://");
```

Comment:
I was thinking about that as well, the problem here is that when passing to a `QUrl` an url like `www.google.com` this is the result:

```
qDebug() << url.scheme() << url.host() << url.url();
# "" "" "www.google.com"
url.setScheme("https");
qDebug() << url.scheme() << url.host() << url.url();
# "https" "" "https:www.google.com"
```

Not really what I was expecting.